src/gui/components/navbar/navbar.styles.js

```diff
@@ -0,0 +1,15 @@
+import { css } from 'glamor'
+import { primaryColor } from '../../styles'
+
+const layout = css({
+  display: 'flex',
+  alignItems: 'center',
+})
+
+export const title = css({
+  fontSize: '1.2em',
+  fontWeight: 'bolder',
+  paddingLeft: '10px',
+})
+
+export default css(primaryColor, layout)
```

Comment:
I have the feeling it doesn't scale.
How do we handle conditionnal styling if you export a wrapped `css` there ?

Can we do : `css(conditionnal, css(primaryColor, layout))` ?

Example : a primary button is blue, a secondary is green. How do we handle that with this default export ?